printer_mods/geoffreyyoung/spider_bracket/README.md

```diff
@@ -1,17 +1,3 @@
 # FYSETC SPIDER DIN Bracket
 
-Minimalist DIN bracket for the [FYSETC SPIDER](https://wiki.fysetc.com/Spider/), based on the simple design of the core SKR bracket.  It uses a [standard build `pcb_din_clip_x3.stl`](https://github.com/VoronDesign/Voron-2/blob/Voron2.4/STLs/VORON2.4/Electronics_Compartment/DIN_Brackets/pcb_din_clip_x3.stl), and provides integrated horizontal and vertical mount options so you can choose on the fly.
-
-## Printing
-
-Use the standard Voron 2.4 recommended print settings.
-
-## BOM
-
-* 4 x M3x8 SHCS screws
-* 1 x [`pcb_din_clip_x3.stl`](https://github.com/VoronDesign/Voron-2/blob/Voron2.4/STLs/VORON2.4/Electronics_Compartment/DIN_Brackets/pcb_din_clip_x3.stl)
-* 2 x whatever hardware you use to mount your other `pcb_din_clip_x3.stl` clips (I use M2x6 screws, but the BOM self-tapping screws work as well).
-
-## Images
-
-![spider_bracket.png](images/spider_bracket.png)
+You can now use the Spider bracket that is included as part of the [Voron Trident](https://github.com/VoronDesign/Voron-Trident) repo, which you can find [here](https://github.com/VoronDesign/Voron-Trident/tree/main/STLs/ElectronicsBay/Controller_Mounts)
```

Comment:
can you change this link to the commit instead of the branch? just in case the clip gets updated and no longer fits your mod at least the link will always be to the correct part.

Reply:
done :)